js/ui/form/ui.form.layout_manager.js

```diff
@@ -773,23 +774,32 @@ var LayoutManager = Widget.inherit({
                 editorInstance = that._createComponent($editor, renderOptions.editorType, editorOptions);
                 editorInstance.setAria("describedby", renderOptions.helpID);
                 editorInstance.setAria("required", renderOptions.isRequired);
+                that._registerEditorInstance(editorInstance, renderOptions);
 
                 if(renderOptions.dataField) {
-                    var componentOwner = that.option("form") || that;
-
-                    editorInstance.on("enterKey", function(args) {
-                        componentOwner._createActionByOption("onEditorEnterKey")(extend(args, { dataField: renderOptions.dataField }));
-                    });
-                    that._registerEditorInstance(editorInstance, renderOptions.dataField);
-                    that._createWatcher(editorInstance, $container, renderOptions);
-                    that.linkEditorToDataField(editorInstance, renderOptions.dataField, renderOptions.editorType);
+                    that._bindDataField(editorInstance, renderOptions, $container);
                 }
             } catch(e) {
                 errors.log("E1035", e.message);
             }
         }
     },
 
+    _getComponentOwner: function() {
+        return this.option("form") || this;
+    },
+
+    _bindDataField: function(editorInstance, renderOptions, $container) {
+        var componentOwner = this._getComponentOwner();
```

Comment:
The `getComponentOwner `is used only once. Maybe will be better use `this.option("form") || this;` instead of method.